L/libsingular_julia/build_tarballs.jl

```diff
@@ -9,15 +9,14 @@ uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
 delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
 
 name = "libsingular_julia"
-version = v"0.34.0"
+version = v"0.35.0"
 
 # reminder: change the above version if restricting the supported julia versions
-julia_versions = [v"1.6.3", v"1.7", v"1.8", v"1.9", v"1.10"]
-julia_compat = join("~" .* string.(getfield.(julia_versions, :major)) .* "." .* string.(getfield.(julia_versions, :minor)), ", ")
```

Comment:
@benlorenz @giordano do you agree with me that `julia_compat` can just be set to "1.6" these days? Or is there something I am missing / forgetting in play here?

Reply:
OK, so one thing I missed was the fact that this also sets an upper bound right now. And yeah, in the future we may want to have "gaps" in the supported versions... although I wonder what the motivation for that would be? The only I can think of is saving CPU and storage resources for Yggdrasil/Buildkite/GitHub?